docs/security-considerations.rst

```diff
@@ -28,9 +28,16 @@ As always, with open source documentation, please help us extend this section
 (especially, some examples would not hurt)!
 
 ********
-Pitfalls
+Pitfalls and Vulnerabilities
 ********
 
+The most comprehensive to date categorization of know vulnerabilities and pitfalls embodies different aspects of development with Solidity: not only the pitfalls of the language itlself, but also the EVM and blockchain levels. The following table summarizes the most important pitfalls and provides references to their description in this and other surces, together with an example of bad a good practice/solution. It also serves as a taxonomy reference for future findings.
```

Comment:
Please only put that paragraph in once the table is in place. The documentation is always live.
Having said that, I'm not sure a table is the right medium for the documentation. You should read about everything anyway and not just take a glance over it.